students/biegon_piotr/lesson_03_functions/the_collatz_sequence.py

```diff
@@ -0,0 +1,18 @@
+print("The Collatz Sequence\n")
```

Comment:
don't need to commit collatz_sequence, when there is better version with input validation :)